lib/modules/manager/paket/parsers/dependencies-file.ts

```diff
@@ -0,0 +1,52 @@
+interface DependenciesFile {
+  groups: DependenciesGroup[];
+}
+interface DependenciesGroup {
+  groupName: string;
+  nugetPackages: Package[];
+}
+interface Package {
+  name: string;
+  options: string[];
+}
+interface ReduceState {
+  groups: DependenciesGroup[];
+  currentGroup: DependenciesGroup;
+}
+
+export function parse(content: string): DependenciesFile {
+  const result: ReduceState = content.split('\n').reduce(
+    (state: ReduceState, line: string) => {
+      const lineParts = line.split(/\s+/).filter((p) => p !== '');
```

Comment:
And import `isNonEmptyString` from `@sindresorhus/is`
```suggestion
      const lineParts = line.split(/\s+/).filter(isNonEmptyString);
```